content/20.pharmaceuticals.md

```diff
@@ -456,6 +467,7 @@ Because the second phase of COVID-19 infection, associated with a severe inflamm
 Once again, this approach does not require the development of compounds specifically designed for the emergent viral threat, but rather hinges on a theoretical understanding of how the virus interacts with a human host and how pharmaceuticals can be used to modify those interactions.
 In contrast, other mAbs can be designed based on a specific pathogen.
 COVID-19 has demonstrated that this development can progress rapidly, with two mAb cocktails currently approved in the United States.
+<!-- Do we have any broad insights into the types of trials that have been run and the degree of clinical evidence they provide? It seems as if in several cases there were small initial trials (some observational or underpowered) that didn't provide strong evidence for or against a drug. We had to wait for RECOVERY, Solidarity, etc. results to draw better conclusions. Is that accurate or worth commenting on? -->
```

Comment:
I'll work on opening a PR to address this and the comment below!

Reply:
Great!  We can leave these comments here for now and resolve them in a future PR after merging this one, which already touches many files.

Reply:
The more I think about this comment, the more I think this is really the crux of the paper.